src/main/webapp/createPlaceGuide.html

```diff
@@ -28,7 +28,7 @@ <h2 class="primary-text align-center">New Audio Guide</h2>
             <input type="hidden" id="placeId" name="placeId">
 
             <label id= "nameInput" class="mdc-text-field mdc-text-field--outlined form-field">
-                <input type="text" class="mdc-text-field__input" aria-labelledby="my-label-id" name = "name" required id = "name" pattern = "[A-Za-z':-\s]+" title = "Enter appropriate name!" maxlength = "20">
+                <input type="text" class="mdc-text-field__input" aria-labelledby="my-label-id" name = "name" required id = "name" pattern = "[A-Za-z\s]+" title = "Enter appropriate name!" maxlength = "20">
```

Comment:
Why did you remove those characters? The name is actually a title of the guide, restricting it to letters only seems too strict.